definitions/tools/bam_readcount.cwl

```diff
@@ -6,10 +6,12 @@ label: "run bam-readcount"
 
 baseCommand: ["/usr/bin/python", "/usr/bin/bam_readcount_helper.py"]
 requirements:
+    - class: ShellCommandRequirement
     - class: DockerRequirement
-      dockerPull: "mgibio/bam_readcount_helper-cwl:1.0.0"
+      dockerPull: "registry.gsc.wustl.edu/fdu/bam_readcount-prefix"
```

Comment:
Changing to an image hosted on the internal registry breaks all pipelines that use this tool for external users, so that seems not great.  Also, we should probably have a review of the changes to that bam_readcount_helper.py tool that you incorporated.

Reply:
In this PR comment, I mentioned: 
>Note that the docker image will be "mgibio/bam_readcount_helper-cwl:1.1.0" in >bam_readcount.cwl once the PR genome/docker-bam_readcount_helper-cwl#3 is merged.